lib/matplotlib/backends/__init__.py

```diff
@@ -26,30 +29,31 @@ def pylab_setup(backend=None):
         The module which contains the backend of choice
 
     new_figure_manager : function
-        Create a new figure manage (roughly maps to GUI window)
+        Create a new figure manager (roughly maps to GUI window)
 
     draw_if_interactive : function
         Redraw the current figure if pyplot is interactive
 
     show : function
-        Show (and possible block) any unshown figures.
+        Show (and possibly block) any unshown figures.
 
     '''
     # Import the requested backend into a generic module object
-    if backend is None:
-        backend = matplotlib.get_backend()  # validates, to match all_backends
-
-    if backend.startswith('module://'):
-        backend_name = backend[9:]
+    if name is None:
```

Comment:
I think 

```python 
global backend
if name is None:
    name = mpl.get_backend()
backend = name
```

will do the right thing here.  It will always defer to an explicitly passed input and then fall back to `get_backend` which in the end just consults rcparams for the requested backend.  Assigning to the variable marked global will change the object that the module-level attribute points at (with out the global it would just bind a local variable).